charts/timed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.13.2
+version: 0.13.3
```

Comment:
following conventional commits the bump should be higher for a feature, but as I did approve the last change and did not bother, I'm doing here aswell